anesthetic/read/cobayamcmcreader.py

```diff
@@ -1,57 +0,0 @@
-"""Tools for reading from getdist chains files."""
-import glob
-import numpy as np
-from anesthetic.read.chainreader import ChainReader
-try:
-    from getdist import loadMCSamples
-except ImportError:
-    pass
-
-
-class CobayaMCMCReader(ChainReader):
-    """Read Cobaya yaml files."""
-
-    def paramnames(self):
-        r"""Read header of <root>.1.txt to infer the paramnames.
-
-        This is the data file of the first chain. It should have as many
-        columns as there are parameters (sampled and derived) plus an
-        additional two corresponding to the weights (first column) and the
-        logposterior (second column). The first line should start with a # and
-        should list the parameter names corresponding to the columns. This
-        will be used as label in the pandas array. Uses GetDist's
-        `loadMCSamples` to infer the tex dictionary.
-        """
-        try:
-            with open(self.root + ".1.txt") as f:
-                header = f.readline()[1:]
-                paramnames = header.split()[2:]
-                s = loadMCSamples(file_root=self.root)
-                tex = {p.name: '$' + p.label + '$' for p in s.paramNames.names}
-                return paramnames, tex
```

Comment:
Currently `read_cobaya` does not read in any TEX data. Was that dropped because of the upcoming #215?

Reply:
Best not to presume -- although I am now making good progress again with #215 (I may be able to get the original idea to work).

I didn't particularly like the dependency on getdist for file reading (ideally we'd do it natively), but you're right that this is an unnecessary removal of functionality. Restored and tested